libs/src/api/account.js

```diff
@@ -610,27 +615,45 @@ class Account extends Base {
       userId = user.userId
     }
 
+    if (!(userId instanceof BN)) {
+      userId = new BN(userId)
+    }
     // matches format for PDA derivation seed in SOL program
-    const userIdSeed = userId.toBuffer('le', 4)
-
-    const {
-      derivedAddress: userAccountPDA
-    } = await this.solanaWeb3Manager.findDerivedPair(
-      this.solanaWeb3Manager.audiusDataProgramId,
-      this.solanaWeb3Manager.audiusDataAdminStorageKeypairPublicKey,
-      userIdSeed
-    )
+    // use BN.toArrayLike instead of .toBuffer for browser compat reasons
+    const userIdSeed = userId.toArrayLike(Buffer, 'le', 4)
+
+    const { derivedAddress: userAccountPDA } =
+      await this.solanaWeb3Manager.findDerivedPair(
+        this.solanaWeb3Manager.audiusDataProgramId,
+        this.solanaWeb3Manager.audiusDataAdminStorageKeypairPublicKey,
+        userIdSeed
+      )
 
     const account = await this.solanaWeb3Manager.fetchAccount(userAccountPDA)
+    return account
```

Comment:
should this deserialize into the User account format?

Reply:
We can do that if it's a req! I wasn't too clear on how this is intended to be consumed downstream by other services and what output format they would expect for this method